src/Features/Core/Portable/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeAction.cs

```diff
@@ -155,9 +155,10 @@ private async Task AddOperatorsAsync(List<IMethodSymbol> members, CancellationTo
 
                 var generator = _document.GetLanguageService<SyntaxGenerator>();
 
+                // add nullable annotation to the parameter reference type, so the value could be null
```

Comment:
This comment is telling us what the code is doing, but not really why. 

Needs explanation related to the fact that (in)equality operator implementations for reference types need to allow comparison against null.